images/runtime/node/14/bullseye.Dockerfile

```diff
@@ -0,0 +1,30 @@
+ARG DEBIAN_FLAVOR
```

Comment:
Similar comment to the `dotnetcore` command above: we should be able to remove these node version specific Dockerfiles and added the following to the `.gitignore`:

```
images/runtime/node/*/bullseye.Dockerfile
```